app/scripts/views/header.coffee

```diff
@@ -126,9 +128,17 @@ module.exports = class HeaderView extends View
     e.preventDefault()
     ViewHelpers.toggleAnonymousStatus()
 
+  SetRefreshAnonymousStatusInterval: ->
+    # Check anonymous status every 500ms and indicate it with icon
+    if ViewHelpers.isLogged() and not @RefreshAnonymousStatusInterval
+      @RefreshAnonymousStatusInterval = setInterval @RefreshAnonymousStatus, 500
```

Comment:
Looks good. Though I think 500ms is too small interval. E.g. if you have 10 tabs with meow opened, it would mean 20 updates per second.